pkg/dbtoken/server.go

```diff
@@ -122,14 +121,6 @@ func (s *server) Run(ctx context.Context) error {
 	return srv.Serve(s.l)
 }
 
-func (s *server) unauthenticatedRoutes(r *mux.Router) {
-	r.NewRoute().Methods(http.MethodGet).Path("/healthz/ready").HandlerFunc(func(http.ResponseWriter, *http.Request) {})
```

Comment:
Previously we only allowed a `GET` request on this path, and no we accept anything.  I know it technically doesn't matter since the endpoint is just returning a 200 status code, but to maintain consistency across the removal should we have a filter on the health probe endpoint to only accept HTTP GET requests in case we make future changes to the health endpoint?

Reply:
I don't see any problems with it, although it is unlikely imho. I implemented it